data/remnant missions.txt

```diff
@@ -495,3 +560,772 @@ mission "Remnant: Technology Available"
+			`	"Congratulations, Captain <last>!" she signs. "Your fluency is good enough that even those unwilling to sing with a stranger should be comfortable working with you. At this rate you will be one of us in more ways that just by blood. I look forward to seeing what results from your work with us." She is interrupted by the arrival of other Remnant asking her something, and she bids you farewell with an apologetic gesture.`
+
+
+
+mission "Remnant: Salvage 1"
+	name "Remnant History Updates"
+	description `Find a copy of human history for the Remnant.`
+	minor
+	source "Viminal"
+	stopover "Alexandria"
+	to offer
+		has "Remnant: Learn Sign 2: done"
+		random < 50
+		or
+			has "Remnant: Catalytic Ramscoop: done"
+			has "Remnant: Plasma Cannons: done"
+			has "Remnant: Heavy Laser: done"
+	on offer
+		conversation
+			`As you step out into the icy chill of Viminal, you get a message from Tali, the engineering prefect, asking if you could meet her in the cafeteria.`
+			choice
+				`	(Not right now.)`
+					defer
+				`	(Accept her invitation.)`
+			`	Replying that you can, you make your way through the crowds and find an empty table. Moments later Tali appears and grabs two trays from the dispenser before heading towards you. She settles down in the opposite chair with a look of exhaustion.`
+			`	"It is good to see you again, <first>," she sings softly. When you sign a reply she sits up and smiles. "My, you are a quick learner," she signs. "Great, this is much faster."`
+			`	"I haven't fully..." (She makes a sign that you don't understand, but seems similar to the signs for "break" and "understand".) "...the technology you retrieved for us, but we should be ready to produce them soon. In the meantime, though, they are showing us that humanity has made significant progress since we departed. Could you help us catch up with what has happened out there?"`
+			choice
+				`	"Yes. What can I do to help?"`
+				`	"Sorry, not interested."`
+					decline
+			branch library
+				has "Human Cultural Archives: done"
+			`	"Excellent! According to a friend of mine who is versed in history from the war, there were rumors that a grand library was being built in the Deep to house all of human knowledge. If it exists, that would likely be where to go."`
+				accept
+			label library
+			`	"Excellent! According to a friend of mine who is versed in history from the war, there were rumors that a grand library was being built in the Deep to house all of human knowledge. If it exists..." she stops when she notices you nodding with a knowing expression. "You have already been there? Well, you certainly get around, captain <last>. I suppose that makes things rather straight forward."`
```

Comment:
"She stops" should be capitalized following the ellipsis (end of a sentence).

Reply:
Changed.